content/en/docs/concepts/cluster-administration/kubelet-garbage-collection.md

```diff
@@ -1,13 +1,13 @@
 ---
 reviewers:
-title: Configuring kubelet Garbage Collection
+title: Garbage collection for container images
 content_type: concept
 weight: 70
 ---
 
 <!-- overview -->
 
-Garbage collection is a helpful function of kubelet that will clean up unused images and unused containers. Kubelet will perform garbage collection for containers every minute and garbage collection for images every five minutes.
+Garbage collection is a helpful function of kubelet that will clean up unused [images](docs/concepts/containers/#container-images) and unused [containers](docs/concepts/containers/). Kubelet will perform garbage collection for containers every minute and garbage collection for images every five minutes.
```

Comment:
Hi @Kartik494 .
Would you add a forward slash before the link path:
`/docs/concepts/containers/`

Reply:
Hi @kbhawkey i have addressed the comment as you suggested , but its again give the link address `https://github.com/kubernetes/website/blob/master/docs/concepts/containers/#container-images` not expected as k8s.io domain , could you please suggest!